flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/AbstractStreamOperator.java

```diff
@@ -208,13 +208,13 @@ public MetricGroup getMetricGroup() {
 	}
 
 	@Override
-	public final void initializeState(OperatorStateHandles stateHandles) throws Exception {
+	public final void initializeState(OperatorSubtaskState stateHandles) throws Exception {
 
 		Collection<KeyedStateHandle> keyedStateHandlesRaw = null;
 		Collection<OperatorStateHandle> operatorStateHandlesRaw = null;
 		Collection<OperatorStateHandle> operatorStateHandlesBackend = null;
 
-		boolean restoring = null != stateHandles;
+		boolean restoring = (null != stateHandles);
```

Comment:
I like to do this when generating a boolean out of a `!=` or `==` comparison because I find this easier to read in the presence of more than one `=` character.